### PR TITLE
Create selfie segmentation

### PR DESCRIPTION
**Pull request by zhibo1-wang**

### CODE REVIEW

**Comment by zhibo1-wang** [comment]
@taste1981 Model is converted from https://github.com/PINTO0309/PINTO_model_zoo/blob/main/082_MediaPipe_Meet_Segmentation, and its license is [LICENSE](https://github.com/PINTO0309/PINTO_model_zoo/blob/main/082_MediaPipe_Meet_Segmentation/LICENSE). And the original model is from MediaPipe, and its license is [Model.Card-Meet.Segmentation.pdf](https://github.com/PINTO0309/PINTO_model_zoo/blob/main/082_MediaPipe_Meet_Segmentation/Model.Card-Meet.Segmentation.pdf). How should I add it?

**Comment by taste1981** [comment]
Please create an .md file linking to the pdf. At the top dir of this repo, create a thirdpartylicense.txt that includes license text of this model, with heading of that license section clarifying that section is for this selfie-segmentation model.

File: selfie-segmentation/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Selfie segmentation
+
+These model is converted from https://github.com/PINTO0309/PINTO_model_zoo/blob/main/082_MediaPipe_Meet_Segmentation, which is from Google MediaPipe. See [Model Card-Meet Segmentation.pdf](https://github.com/PINTO0309/PINTO_model_zoo/blob/main/082_MediaPipe_Meet_Segmentation/Model.Card-Meet.Segmentation.pdf) for original model details.
```
Comment (taste1981) [inline selfie-segmentation/README.md]: are converted from.

File: selfie-segmentation/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+## Input
+
+A tensor of shape 1 x 3 x H x W and data type of 32-bit float ranges from [0, 1], representing a three-channel RGB image of width H and height W whose value is divided by 255.
+
+H=144 and W=256 for the model in 144x256 directory.
+
+## Output
+
+A tensor of shape 1 x H x W and data type of 8-bit unsigned integer ranges from [0, 255], representing the probability of the corresponding pixel belonging to human, multiplied by 255 and truncated into integer.
```
Comment (taste1981) [inline selfie-segmentation/README.md]: ranging from

**Review by taste1981 — APPROVED** [review]

**Pull request merged**